client/layout/store-alerts/style.scss

```diff
@@ -119,6 +119,12 @@
 
 	.components-button {
 		padding: 4px;
+
+		/*!rtl:ignore*/
```

Comment:
This wouldn't be any good for a RTL user that uses a screen reader. I don't know if that's too much of an edge case to worry about though..

Reply:
The buttons (`IconButton` instances) have labels which render as `aria-label` attributes to provide text for screen readers (you can also see this by hovering over the buttons), so I think that would cover the RTL screen reader scenario. Or, is there something else I'm overlooking?